composer.json

```diff
@@ -21,7 +21,9 @@
         "php" : ">=7.1",
         "ext-json": "*",
         "psr/http-client": "^0.3.0",
-        "guzzlehttp/psr7": "^1.4"
+        "guzzlehttp/psr7": "^1.4",
+        "psr/http-factory": "^1.0",
+        "zendframework/zend-diactoros": "^2.0"
```

Comment:
It doesn't make sense to require both `guzzlehttp/psr7` **and** `zendframework/zend-diactoros`.